controller/app/index.js

```diff
@@ -77,21 +82,57 @@ function gotTable(data) {
         return Object.assign({ sched, task }, item)
     }).filter((item) => item.Status == 1 || item.sched.Schedule)
     if (firstLoad) {
-    let stringify = (d) => JSON.stringify(d, null, 2);
-    $("#taskTable").bootstrapTable({
-        idField: 'UUID',
-        columns: [
-            {title:'ID', field:'UUID'},
-            {title:'Task', field:'task', formatter: stringify},
-            {title:'Schedule', field:'sched', formatter: stringify},
-            {title:'Delete', field: 'operate', align: 'center', formatter: operate, events: { 'click .remove': cancel}}
-        ],
-        data: processedData,
+        let stringify = (d) => JSON.stringify(d, null, 2);
+        $("#taskTable").bootstrapTable({
+            idField: 'UUID',
+            columns: [
+                {title:'ID', field:'UUID'},
+                {title:'Task', field:'task', formatter: stringify},
+                {title:'Schedule', field:'sched', formatter: stringify},
+                {title:'Delete', field: 'operate', align: 'center', formatter: operate, events: { 'click .remove': cancel}}
+            ],
+            data: processedData,
+        });
+        firstLoad = false
+    } else {
+        $("#taskTable").bootstrapTable('load', processedData)
+    }
+}
+
+function gotRegions(data) {
+    regionList = data["regions"]
+    $.each(regionList, (reg) => {
+        $('<button/>')
```

Comment:
maybe this is a radio choice? you should be only able to choose 1 of them at a time, right?

Reply:
I think this behaves like a radio. Also it looks nice :)